packages/extension/src/libs/background/internal/get-private-key.ts

```
@@ -0,0 +1,29 @@
+  const password = message.params[1] as string;
+  try {
+    const privKey = await keyring.getPrivateKey(account, password)
+    return {
+      result: JSON.stringify(privKey),
+    };
+  } catch (e: any) {
+    console.error(e)
+    return {
+      error: getCustomError(e.message),
+    };
+  };
```

Biome (1.9.4)

[error] This code is unreachable

... because either this statement, ...

... this statement, ...

... or this statement will return from the function beforehand


(lint/correctness/noUnreachable)